contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.2.0--4.3.0.sql

```diff
@@ -591,6 +591,60 @@ WHERE (SELECT orig_username FROM sys.babelfish_authid_user_ext WHERE rolname = C
 
 GRANT SELECT ON sys.user_token TO PUBLIC;
 
+CREATE OR REPLACE VIEW sys.server_principals
```

Comment:
If we changed the MVU script, then we should add a MVU test for this to verify it works fine with MVU.

Reply:
The basic test has been added there in sys-server_principals